coderd/database/pubsub/latency.go

```diff
@@ -0,0 +1,55 @@
+package pubsub
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"golang.org/x/xerrors"
+)
+
+var channelID uuid.UUID
```

Comment:
global state plus `t.Parallel()` is a recipe for sadness. Let's scope this to the Pubsub.

Reply:
Good point. I've refactored out the global state in https://github.com/coder/coder/pull/13126/commits/28a96dec4a21acd88fc4718a58c7d6e3d9810cde. This makes the code a bit more complex than it needs to be, so I'm tempted to revert this and drop `t.Parallel()` to favour simplicity over a relatively minor CI speedup.

Thoughts?

Reply:
This was addressed in https://github.com/coder/coder/commit/65f57b1cc2842f40fcd5ed4f775e60ce18fb12b7.